### DISCUSSION TITLE
✅ 🔗 Hide hidden content in source code for Google & Co. (Duplicate)

### DISCUSSION THREAD

**Discussion by sonnesworld** [elementor/elementor#14343] [Feature Request]
On some websites it would be helpful if the content is not only visible for user but also not visible for search engines (Google & co.). I created a website for a client with job descriptions and hide them on all devices. So he can easily hide or show different job ads which are already created in backend. But when this content is hide bots can read this job descriptions which are not visible for website visitors. Some external platforms crawl this website and show job ads which are hidden. With Elementor the content is still visible in page source code, even I hide it in backend.

Some screenshots:
https://drive.google.com/file/d/1zbwHMuqvoSgzpuqlnxeRWdLsJT_brfgk/view?usp=sharing
https://drive.google.com/file/d/1FyPToYdP3JYDllcAfCrL9CcQRqfdfHSM/view?usp=sharing

**Comment by nicholaszein** [comment]
Hello there @sonnesworld!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the introduction of the widget Display Conditions in Elementor Pro v3.19! 🥳 

For more information check this article: https://elementor.com/help/show-and-hide-elements-on-a-page-display-conditions/

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
